docs/mkdocs.yml

```diff
@@ -1008,7 +1008,7 @@ markdown_extensions:
 extra:
   analytics:
     provider: google
-    property: UA-110480857-1
+    property: G-DH3T88LK5K
```

Comment:
We should verify if this is sufficient. Here's the full snippet that we should embed following GA guidance:

```html
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-DH3T88LK5K"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-DH3T88LK5K');
</script>
```

If I look at the current implementation, it refers to a `ga()` function. Before we apply this changes, let's double check that the implementation in the current theme can handle the GA4 tag so that we don't lose tracking data.

Reply:
According to the docs of `mkdocs` this should be sufficient. https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#google-analytics

![image](https://user-images.githubusercontent.com/11723921/235318379-07be804a-e379-40e4-946f-e6ff1e72c692.png)

This has been added in the `v7.1.8` release and we are using the `v8.5.3`. I'm also not sure if there's a way to test this locally. Most SSGs disable analytics when running locally so your data is not polluted.

Reply:
We could verify by deploying to a copy of the repo. On the other hand, if it's mentioned explicitly in the docs, then we can rely on it and just roll with it. Worst case, we revert it.